.env.dev.example

```diff
@@ -0,0 +1,32 @@
+# production.rb OVERRIDES
+KLAXON_FORCE_SSL=false # IRL this should be true, but for local "prod" needs to be false
+KLAXON_COMPILE_ASSETS=true
+
+# App settings
+# add emails that you want to be able to authenticate as admin
+ADMIN_EMAILS="admin@news.org, other.admin@news.org"
+# port is likely 5432, db name is likely something like "klaxon"
+# cluster endpoint can be found in the RDS cluster: 
+# https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/accessing-monitoring.html
```

Comment:
No, I think this is just the ticket! The idea would be that our main contribution back improves the flow for AWS deployment. I think it would be better to leave to other engineers the contributions that weave in Heroku deployment